docs/sections/user_guide/yaml/components/make_hgrid.rst

```diff
@@ -0,0 +1,252 @@
+
+simple_dy:
+""""""""""
+
+Specify the uniform cell length in y-direction for simple cartesian grid.
+
+grid_name:
+""""""""""
+
+Specify the grid name.
+
+center:
+"""""""
+
+Specify the center location of grid.
+
+shift_fac:
+""""""""""
+
+Shift west by 180/shift_fac. Default value is 18.
+
+do_schmidt:
+"""""""""""
+
+Set to do Schmidt transformation to create astretched grid. When do_schmidt is set, the following must be set: --stretch_factor, --target_lon and --target_lat.
+
+do_cube_transform:
+""""""""""""""""""
+
+Set to re-orient the rotated cubed sphere so that tile 6 has 'north' facing upward. When do_cube_transform is set, the following must be set: --stretch_factor, --latget_lon, and --target_lat.  
+
+stretch_factor:
+"""""""""""""""
+
+Stretching factor for the grid.
+
+target_lon:
+"""""""""""
+
+Center longitude of the highest resolution tile.
+
+target_lat:
+"""""""""""
+
+Center latitude of the highest resolution tile.
+
+nest_grids:
+"""""""""""
+
+Set to create this # nested grids as well as the global grid.
+
+parent_tile:
+""""""""""""
+
+Specify the list of the parent tile number(s) of nest grid(s).
+
+refine_ratio:
+"""""""""""""
+
+Specify the list of refinement ratio(s) for nest grid(s).
+
+halo:
+"""""
+
+Halo size is to make sure the nest, including the halo region, is fully contained within a single parent (coarse) tile.
+
+istart_nest:
+""""""""""""
+
+Specify the list of starting i-direction index(es) of nest grid(s) in parent tile supergrid(Fortran index).
+
+iend_nest:
+""""""""""
+
+Specify the list of ending i-direction index(es) of nest grids in parent tile supergrid(Fortran index).
+
+jstart_nest:
+""""""""""""
+
+Specify the list of starting j-direction index(es) of nest grids in parent tile supergrid(Fortran index). 
+
+jend_nest:
+""""""""""
+
+Specify the list of ending j-direction index(es) of nest grids in parent tile supergrid(Fortran index).
+
+great_circle_algorithm:
+"""""""""""""""""""""""
+
+When specified, great_circle_algorithm will be used to compute grid cell area. 
+
+out_halo:
+"""""""""
+
+Extra halo size data to be written out.
+
+angular_midpoint:
+"""""""""""""""""
+
+If specified, when grid_type is from_file and the input is a NetCDF file, then the supergrid face midpoint coordinates are simply and independely (the lat independently from the lon) calculated as simple angular midpoints from the model grid coordiantes.
+
+non_length_angle:
+"""""""""""""""""
+
+When specified, will not output length(dx,dy) and angle (angle_dx, angle_dy).
+
+rotate_poly:
+""""""""""""
+
+Set to calculate polar polygon areas by calculating the area of a copy of the polygon, with the copy  being rotated far away from the pole.
+
+verbose:
+""""""""
+
+Will print out running time message when this is set. Otherwise the run will be silent when there is no error.
+
+Example UW YAML With SRW YAML References
+----------------------------------------
```

Comment:
I would prefer we leave this section out. The point of a portion of our work is to refactor how SRW does this, in which case none of these environment variable will exist, and none of our tools support the syntax used here.

Reply:
Thats fair - I was torn between that, and wanting to help users with the remapping thats not necessarily clear without deeper references

Reply:
I've kept it as a note if needed later and removed it from here